core/templates/services/speech-synthesis-chunker.service.ts

```diff
@@ -28,6 +28,8 @@ import { downgradeInjectable } from '@angular/upgrade/static';
 
 import { ServicesConstants } from 'services/services.constants.ts';
 
+import MathLive from 'mathlive';
```

Comment:
How big is this library? (after minification and gzip) Did you consider any alternative libraries?

Reply:
mathlive.js file has 438.5 kB

Reply:
You can check it through here for example https://www.giftofspeed.com/gzip-test/ (and use https://raw.githubusercontent.com/arnog/mathlive/master/dist/mathlive.js). 100 KB is still quite big did you check if there exist any alternative libraries?